public/data/Sahilll15.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Sahil Sanjay Chalke",
+  "location": "Mumbai, India 🇮🇳", 
+  "bio": "FULL STACK DEV",
+  "avatar": "https://avatars.githubusercontent.com/u/109215419?v=4",
+  "portfolio": "https://sahilchalke.online/",
+  "skills": [
+    "Javascript",
+    "C++",
+    "Typescript",
+    "Nodejs",
+    "Expressjs",
+    "React",
+    "Next.Js",
+    "MySQL",
+    "MongoDB",
+    "Docker",
+    "Git",
+    "Tailwind CSS",
+    "REST APIs",
+    "GraphQL",
```

```suggestion
    "GraphQL"
```

@Sahilll15 Remove the comma from last element.